workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -5,7 +5,8 @@ nextflow.enable.dsl=2
 params.ref_dir = 's3://nextflow-ccdl-data/reference/homo_sapiens/ensembl-100'
 params.cdna = 'fasta/Homo_sapiens.GRCh38.cdna.all.fa.gz'
 params.txome = 'fasta/Homo_sapiens.GRCh38.txome.fa.gz'
-params.kmer = [23, 31]
+params.genome = 'fasta/Homo_sapiens.GRCh38.dna.primary_assembly.fa.gz'
+params.kmer = '31'
```

Comment:
I assume this change is intentional because only k = 31 full SA indexes are over on the benchmarking PR (#18). How does this change fit into the index generation big picture? Specifically, would this change how I would run `build-index.nf` if I still wanted to use k = 23 and I didn't have a local copy of those yet?

Reply:
This was intentional, yes, partly because initial benchmarking indicates that the k = 31 is better (faster, no lower mapping rate), but also because I found that `params` can't really be lists. Obviously, it works here, but there is no standard way to specify a list at the command line to override these parameters. 

To run with a different `params.kmer` value, you can run:
````
nextflow build-index.nf --kmer 23
```